mavedb_mapping/test_for_invalid_inputs.py

```diff
@@ -0,0 +1,21 @@
+from main import main_map
+import pytest
+
+
+class TestForNonHumanScoresets:
+    def test_for_scoreset_if_blat_successful(self):
```

Comment:
add a docstring here, indicating that we expect failure for non-human target scoresets

Reply:
@afrubin, I was discussing with @samriddhi99, do we want non-human targets to be in-scope for this project?

Reply:
We should revisit non-human targets once this is well supported by biocommons.